### DISCUSSION TITLE
Do you need multiple cloudflare workers for multiple domains or can they all use the same worker? (Cloudflare proxy)

### DISCUSSION THREAD

**Discussion by Cdingram** [plausible/analytics#2835] [Anything Else]
Just as the title states, do you need multiple cloudflare workers for multiple domains or can they all use the same worker? I'm guessing as long as each site has the proper data-domain in the header script they can use the same worker but wan't to confirm there would be no cross contamination.

**Comment by metmarkosaric** [comment]
hi @Cdingram! yes, you're right. you can use the same cloudflare worker for more sites. just make sure to have the correct data-domain in the snippet so the stats get recorded to the correct dashboard

**Reply by Cdingram** [reply]
Awesome, appreciate the help!
